src/settings/Models/FinePolicy/utils/helper.test.js

```diff
@@ -0,0 +1,23 @@
+import helpers from './helpers';
+
+describe('FinePolicy helpers', () => {
```

Comment:
```
describe('FinePolicy helpers', () => {
    const options = [
      { id: 'selector1' },
      { id: 'selector2'},
    ];

  it('"isValidItemSelected" should return true if some matches', () => {
    const selector= 'selector1';

    expect(helpers.isValidItemSelected(options, selector)).toEqual(true);
Stanislau-Snapok marked this conversation as resolved.
  });

  it('"isValidItemSelected" should return false if there is no matches', () => {
    const selector= 'selector3';

    expect(helpers.isValidItemSelected(options, selector)).toEqual(false);
  });
});
```

Reply:
Can we move options to reusable part of tests?

Reply:
Fixed